editor/test/integration/build_test.clj

```diff
@@ -779,11 +779,10 @@
                               ["more_assets/some_more.stuff" "some_more.stuff"]
                               ["more_assets/some_more2.stuff" "some_more2.stuff"]]))
       (with-setting "project/custom_resources" "nonexistent_path"
-        (project-build project game-project (g/make-evaluation-context))
-        (doseq [path ["assets/some.stuff" "assets/some2.stuff"
-                      "root.stuff"
-                      "more_assets/some_more.stuff" "more_assets/some_more2.stuff"]]
-          (is (false? (.exists (build-path workspace path)))))))))))
+        (let [build-error (:error (project-build project game-project (g/make-evaluation-context)))
+              error-message (some :message (tree-seq :causes :causes build-error))]
+          (is (g/error? build-error))
+          (is (= "Custom resource not found: '/nonexistent_path'" error-message)))))))))
```

Comment:
While the additional test of the error message is nice, it looks to me like the original test was checking that the output files from the previous build were cleaned up correctly. I don't quite know why that test failed, but I don't think we can simply ignore it and remove the check like this.

Reply:
Sorry. I'm still struggling to understand Clojure (plus it was late at night with a fried brain). I'll investigate.

Reply:
I think what's happening here is normal since the build fails before it gets to cleanup the output dir. Question is: should we keep it this way (and also keep the previous test, but with a non-failing build) (I just amended my last commit with this change), or change the behaviour of the build process in some way?

Reply:
I think you're right, that makes sense. Thanks for looking into it!

I think we should merge it like this. @eangelin do you have any reservations about the changed test?